src/components/StatsUsageChart.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <base-line-chart :data-sets="dataSets" :labels="labels" :key="season.id" />
+</template>
+
+<script>
+import BaseLineChart from '@/components/BaseLineChart.vue';
+
+export default {
+  name: 'StatsUsageChart',
+  components: {
+    BaseLineChart,
+  },
+  props: {
+    season: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    theme() {
+      return this.$vuetify.theme.themes.cuttleTheme.colors;
+    },
+    labels() {
+      return this.season.gameCounts.map((_el, index) => `Week ${index + 1}`);
```

Comment:
Labels on the X axis go 'Week 1', 'Week 2', etc for as many weeks as are included